README.rst

```diff
@@ -18,7 +18,7 @@ This product was developed for use with Plone and Dexterity.
 Requirements
 ------------
 
-* Plone >= 4.3
+* Versions >= 1.3 are for Plone 5+, if you use Plone 4.3, use versions 1.2.x
```

Comment:
Should this be added to 1.2x branch as well? https://github.com/collective/collective.z3cform.datagridfield/blob/9b915e3b524672c2abbda34b4203670a46e74ffd/setup.py#L26 `Products.CMFPlone < 5`?

Reply:
You're welcome to do that